client/blocks/image-editor/index.jsx

```diff
@@ -225,6 +223,7 @@ const ImageEditor = React.createClass( {
 			<div className={ classes }>
 				{ this.state.canvasError && this.renderError() }
 
+				<CloseOnEscape onEscape={ this.onCancel } />
```

Comment:
I'm aware that it isn't the primary purpose of this PR but could we rename this method defining what it will do if something happens?
When `onCancel` happens then `cancelImageEdition` or something like that.